src/queue.ts

```diff
@@ -1124,10 +1128,28 @@ const queue = {
             "%)",
           ""
         );
```

Comment:
We should probably use `i18next` for the list of levels for 2 reasons:
Some languages can use different brackets like Japanese for example.
Some languages can use different separators of items:
https://linguistics.stackexchange.com/questions/6737/what-languages-use-the-serial-comma#answer-6791

Maybe we should just use `Intl.ListFormat` for formatting lists in general (also for the other places where lists are printed like custom codes and custom levels).
See https://www.i18next.com/translation-function/formatting#list
And then also add a translation entry for the `{{submitter}} ({{percentage}}%)` or something like this.